.circleci/config.yml

```diff
@@ -0,0 +1,242 @@
+version: 2.1
+
+orbs:
+  android: circleci/android@0.2.1
+  react-native: react-native-community/react-native@4.4.2
+
+commands:
+
+  build-flutter-project:
+    description: "Install dependencies and build"
+    parameters:
+      working_directory:
+        default: .
+        type: string
+    steps:  
+      - run:
+          name: Install Dependencies
+          command: flutter pub get
+      - run:
+          name: Build
+          command: flutter packages pub run build_runner build
+          
+  run-flutter-integration-tests:
+    description: "Run Flutter integration tests"
+    steps:
+      - run:
+          name: Run tests
+          command: |
+            cd example
+            flutter drive --dart-define=API_KEY=$API_KEY --target=integration_test/app_test.dart \
+            --driver=test_driver/integration_test.dart
+
+  install-and-build-flutter:
+    description: "Install and build Flutter"
```

Comment:
this command seems fundamentally the same as `build-flutter-project`, but with a `cd example`. perhaps they can be unified?

Reply:
yes, good catch, I realized that and changed it in 6dc9ba5995f5e4d1a225fbb42582aa2cb32576ba